google-domains

```diff
@@ -7122,3 +7122,458 @@
 0.0.0.0 www.wiki.fitbit.com
 0.0.0.0 www.windchill.fitbit.com
 0.0.0.0 wwwinternal.fitbit.com
+
+
+
+0.0.0.0 stats.lineageos.org
```

Comment:
It would be telemetry as:

0.0.0.0 acceleratedmobilepageurl.googleapis.com
0.0.0.0 androidauto-pa.googleapis.com
0.0.0.0 android.bugly.qq.com
0.0.0.0 app-measurement.com # Most common Google tracker
0.0.0.0 app.adaway.org # Adaway autoupdate; not necessary
0.0.0.0 appspot.com
0.0.0.0 autofill.googleapis.com
0.0.0.0 bugly.qq.com
0.0.0.0 bugsnag.com
0.0.0.0 crashlytics.com
0.0.0.0 dataprep.googleapis.com
0.0.0.0 discover-pa.googleapis.com
0.0.0.0 firebaseinstallations.googleapis.com
0.0.0.0 firebase-settings.crashlytics.com
0.0.0.0 piwik.darken.eu # SD Maid
0.0.0.0 privatestats.whatsapp.net # Most important WhatsApp tracker
0.0.0.0 reports.crashlytics.com
0.0.0.0 sessions.bugsnag.com
0.0.0.0 settings.crashlytics.com
0.0.0.0 stats.lineageos.org
0.0.0.0 whitepages.com # Replicant bug, now fixed